src/gfx/main.cpp

```diff
@@ -761,9 +761,16 @@ int main(int argc, char *argv[]) {
 		}());
 		if (options.palSpecType == Options::EXPLICIT) {
 			fputs("\t[\n", stderr);
-			for (std::array<Rgba, 4> const &pal : options.palSpec) {
-				fprintf(stderr, "\t\t#%06x, #%06x, #%06x, #%06x,\n", pal[0].toCSS() >> 8,
-				        pal[1].toCSS() >> 8, pal[2].toCSS() >> 8, pal[3].toCSS() >> 8);
+			for (const auto &pal : options.palSpec) {
+				fputs("\t\t", stderr);
+				for (auto& color : pal) {
+					if (color) {
+						fprintf(stderr, "#%06x, ", color.value().toCSS() >> 8);
```

Comment:
A little conciseness nitpick.

```suggestion
						fprintf(stderr, "#%06x, ", color->toCSS() >> 8);
```